en/docs/install-and-setup/setup/si-setup/si-cicd-k8s.md

```diff
@@ -0,0 +1,189 @@
+
+#### Deployment Descriptor Repository
+- We maintain Deployment Descriptor repositories in Github. 
+- There will be Kubernetes yaml files for each project inside a separate folder.
+- This repository will have three branches representing each deployment environment.
+- To use a new version / rollback to a previous version, the user should define the version inside the respective `deployment.yaml` and commit to the respective branch.
+
+#### Deployment Descriptor Jenkins Build Job
+- We need to maintain one Jenkins job per Environment.
+- This job contains only the build phase. This will apply the relevant yaml files in the relevant environment.
+
+### Kubernetes artifacts
+Sample Kubernetes artifacts for a stateless deployment have been provided below. You can extend the below yaml files to introduce other components. (eg: configMaps, Ingress etc) 
+
+```yaml tab="Deployment"
+apiVersion: "apps/v1"
+kind: Deployment
+metadata:
+  name: "si-deployment"
+spec:
+  replicas: 1
+  minReadySeconds: 30
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      app: "integration"
+  template:
+    metadata:
+      labels:
+        app: "integration"
+    spec:
+      containers:
+      - name: wso2si-sample
+        image: docker.io/samplerepo/helloworld:1.0.0
+        imagePullPolicy: Always
+        resources:
+          requests:
+            memory: "1Gi"
+            cpu: "1000m"
+          limits:
+            memory: "1Gi"
+            cpu: "1000m"
+        ports:
+        -
+          containerPort: 9090
+          protocol: "TCP"
+        -
+          containerPort: 9443
+          protocol: "TCP"
+        -
+          containerPort: 9711
+          protocol: "TCP"
+        -
+          containerPort: 9611
+          protocol: "TCP"
+        -
+          containerPort: 7711
+          protocol: "TCP"
+        -
+          containerPort: 7611
+          protocol: "TCP"
+```
+
+```yaml tab="Service"
+apiVersion: "v1"
+kind: "Service"
+metadata:
+  name: "si-service"
+spec:
+  type: "ClusterIP"
+  selector:
+    app: "integration"
+  ports:
+    # ports that this service should serve on
+    -
+      name: pass-through-http
+      protocol: TCP
+      port: 9090
+    -
+      name: servlet-https
+      protocol: TCP
+      port: 9443
+    -
+      name: http-port-1
+      protocol: TCP
+      port: 9711
+    -
+      name: http-port-2
+      protocol: TCP
+      port: 9611
+    -
+      name: http-port-3
+      protocol: TCP
+      port: 7711
+    -
+      name: http-port-4
+      protocol: TCP
+      port: 7611
+```
+
+    
+## Setting up the environment
+
+### Prerequisites
+
+1. Streaming Integrator Tooling
+
+2. GitHub repository - To maintain the descriptor files of each environment
+
+3. Jenkins server for Continuous Deployment
+
+4. Docker registry - To store the SI images.
+
+5. Kubernetes cluster with 3 different namespaces (dev, staging, prod)
```

Comment:
Should be unordered bullets. We should use numbers only when the order of steps is important/relevant.

Reply:
Fixed